index.rst

```diff
@@ -116,6 +120,9 @@ The conceptual type of a :ref:`DataUnit` (such as visit, tract, or filter).
 
 In the :ref:`Common Schema <CommonSchema>`, each :ref:`DataUnitType` is a table that the holds :ref:`DataUnits <DataUnit>` of that type as its rows.
 
+[KSK] Maybe having the "Common Schema" section fleshed out a little will help with this, but I'm not clear from the discussion what each of these is.  Here's my understanding as a concrete example.  
+
+``DataUnit`` -- 'r' (filter), ``DataUnitType`` -- filter.  Again the word type is causing me confusion the type of the ``DataUnit`` is ``String``, but I get the impression that is supposed to be introspected from the "Common Schema".
```

Comment:
I don't think the fact that the primary identifier for a filter `DataUnit` is a string like "r" is meaningful.  Consider a "sensor" `DataUnitType`: that's a meaningful concept whether you usually identify a particular sensor with an int (as in HSC) or a string like "R11S12" (as in LSST).  The `DataUnit` exists independent of the label(s) for it.

Anyhow, fleshing out the CommonSchema section should help with that.

Reply:
I'm just stumbling over words.  I think, as above, just switching from `DataUnitType` to `DataUnitLabel` or something would help.